packages/headless/src/features/commerce/sort/sort-slice.ts

```diff
@@ -0,0 +1,42 @@
+import {createReducer} from '@reduxjs/toolkit';
+import {applySort} from './sort-actions';
+import {getCommerceSortInitialState} from './sort-state';
+import {fetchProductListing} from '../product-listing/product-listing-actions';
+import {SortOption} from '../../../api/commerce/product-listings/v2/sort';
+import {
+  buildRelevanceSortCriterion,
+  SortBy,
+  SortCriterion,
+} from '../../sort/sort';
+
+const sortResponseToSort = (sort: SortOption): SortCriterion => {
```

Comment:
This name is a bit confusing; it can be read as "sort a response that needs to be sorted" or "convert a sort response to sort"

I'd name it `convertSortOptionToStateCriterion` or `convertResponseSortToStateSort`

Reply:
Renamed it to `mapResponseSortToStateSort`. Let me know if that's good!